### ISSUE TITLE
link to github from collection  

### ISSUE THREAD

**Issue by kcranston** [OpenTreeOfLife/opentree#871]
Should be to item in github repo, raw, e.g. [regular github link](https://github.com/OpenTreeOfLife/collections-1/blob/master/collections-by-owner/kcranston/barnacles.json) and [raw](https://raw.githubusercontent.com/OpenTreeOfLife/collections-1/master/collections-by-owner/kcranston/barnacles.json)

**Comment by kcranston** [comment]
I think this might actually be a peyotl issue, but I am having a hard time following the code. The relevant collection property is [`external_url`](https://github.com/OpenTreeOfLife/opentree/blob/17f429a8f5dc42ba37c7cee7f3f1ce09aa219f73/curator/views/layout.html#L557). This comes from [`get_external_url`](https://github.com/OpenTreeOfLife/peyotl/blob/7086a4a2b7ea242877038366cae5497a00ce7fd7/peyotl/api/collections_api.py#L76) in peyotl, set in [`_remote_external_url`](https://github.com/OpenTreeOfLife/peyotl/blob/7086a4a2b7ea242877038366cae5497a00ce7fd7/peyotl/api/collections_api.py#L192). This URL gets used  in github interactions, so perhaps the solution is a new property for collections? `view_url` or something similar?

**Comment by jimallman** [comment]
Thanks, I'll take a look!

**Comment by jimallman** [comment]
>  perhaps the solution is a new property for collections? view_url or something similar?

Agreed, maybe something like `repo_url` or `source_repository_url`? I could see this being useful for all types in peyotl, actually (alongside `external_url` wherever it appears).

**Comment by jimallman** [comment]
If that's too much, we could munge `external_url` to get to its main page in the repo. Just change this:
`https://raw.githubusercontent.com/OpenTreeOfLife/collections-0/master/`
...to this:
`https://github.com/OpenTreeOfLife/collections-0/blob/master/`

**Comment by kcranston** [comment]
I was checking to see how we handle this with studies, but we don't actually provide a link to the study in github (other than the diff view in the history tab).

**Comment by jimallman** [comment]
See the comments on #964 for alternative solutions, esp. changing the URL we provide, or providing separate "download" and "history" URLs.
